tests/interfaces/default_qubit_2_integration/test_autograd_qnode.py

```diff
@@ -0,0 +1,2030 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for using the autograd interface with a QNode"""
+import autograd
+import autograd.numpy as anp
+import pytest
+
+import numpy as onp
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane import qnode
+from pennylane.tape import QuantumScript
+from pennylane.devices.experimental import DefaultQubit2
+
+qubit_device_and_diff_method = [
```

Comment:
can we remove the creation/parametrization over this many copies of the device? it shouldn't be stateful hopefully. I know it's a copy-paste from the existing file so nbd either way